pkg/operator/backupcontroller/backupcontroller.go

```diff
@@ -144,6 +145,16 @@ func (c *BackupController) sync(ctx context.Context, _ factory.SyncContext) erro
 	})
 
 	klog.V(4).Infof("BackupController backupsToRun: %v, chooses %v", backupsToRun, backupsToRun[0])
+	valid, err := validateBackup(ctx, backupsToRun[0], c.kubeClient, backupsClient)
+	if err != nil {
+		return err
+	}
+
+	if !valid {
+		klog.V(4).Infof("BackupController deems: %v invalid, skipping", backupsToRun[0])
+		return nil
```

Comment:
so if the backup is invalid, we just log it, no err returned 

the next sync() it will be the same, or ?

Reply:
yes, because the backup failure state is set on the CRD, no need to degrade the operator in this case